packages/sublime_merge.rb

```diff
@@ -0,0 +1,41 @@
+require 'package'
+
+class Sublime_merge < Package
+  description 'A snappy UI, three-way merge tool,side-by-side diffs, syntax highlighting, and more.'
```

Comment:
The downside: _"Sublime Merge may be downloaded and evaluated for free, however a license must be purchased for continued use."_

Reply:
Hoping they change their mind about that.